website/client/components/inventory/stable/index.vue

```diff
@@ -0,0 +1,656 @@
+<template lang="pug">
+  .row.stable
+    .standard-sidebar
+      div
+        b-popover(
+          :triggers="['hover']",
+          :placement="'right'"
+        )
+          span(slot="content")
+            h4.popover-content-title {{ $t('mattBoch') }}
+            .popover-content-text {{ $t('mattBochText1') }}
+
+          div.npc_matt
+
+      .form-group
+        input.form-control.input-search(type="text", v-model="searchText", :placeholder="$t('search')")
+
+      .form
+        h2(v-once) {{ $t('filter') }}
+        h3(v-once) {{ $t('pets') }}
+        .form-group
+          .form-check(
+            v-for="petGroup in petGroups",
+            v-if="viewOptions[petGroup.key].animalCount != 0",
+            :key="petGroup.key"
+          )
+            label.custom-control.custom-checkbox
+              input.custom-control-input(type="checkbox", v-model="viewOptions[petGroup.key].selected")
+              span.custom-control-indicator
+              span.custom-control-description(v-once) {{ petGroup.label }}
+        h3(v-once) {{ $t('mounts') }}
+        .form-group
+          .form-check(
+            v-for="mountGroup in mountGroups",
+            v-if="viewOptions[mountGroup.key].animalCount != 0",
+            :key="mountGroup.key"
+          )
+            label.custom-control.custom-checkbox
+              input.custom-control-input(type="checkbox", v-model="viewOptions[mountGroup.key].selected")
+              span.custom-control-indicator
+              span.custom-control-description(v-once) {{ mountGroup.label }}
+
+        div.form-group.clearfix
+          h3.float-left Hide Missing
+          toggle-switch.float-right.hideMissing(
+            :label="''",
+            :checked="hideMissing",
+            @change="updateHideMissing"
+          )
+
+    .standard-page
+      .clearfix
+        h1.float-left.mb-0.page-header(v-once) {{ $t('stable') }}
+        b-dropdown.float-right(:text="$t('sortBy') +((selectedSortBy === 'standard')?'': ': '+ $t(selectedSortBy))", right=true)
```

Comment:
not important but you can use ES6 in templates too, so ``string ${var}`` for example